modules/ossm-migrating-gateways-canary.adoc

```diff
@@ -0,0 +1,84 @@
+
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/migrating/migrating-gateways/ossm-migrating-gateways-assembly.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-migrating-gateways-canary_{context}"]
+= Gateway canary migration
+
+The gateway canary migration scenario is recommend for a gradual rollout. It uses multiple gateway versions with maximum control over gateway rollout.
+
+[NOTE]
+====
+The label for the gateway namespace differs between mulitenant and cluster-wide meshes. Return to the either the multitenant or cluster-wide migration guide to understand which labels to use in your specific migration case, and to see if the `maistra.io/ignore-namespace: "true"` label should be removed.
+====
+
+.Procedure
+
+. Label the gateway namespace to ensure injection from the new mesh is enabled for the namespace.
+
+. Add the `maistra.io/ignore-namespace: "true"` label.
```

Comment:
It's imported to have all steps together, please follow what we have in the eng doc - https://github.com/openshift-service-mesh/sail-operator/blob/main/docs/ossm/ossm2-migration/gateway-migration.md#gateway-migration-guide